pipeline.sh

```diff
@@ -172,10 +192,10 @@ ELAPSED_TIME=$(($SECONDS - $START_TIME))
 echo "step 2/3 filtering completed in $(($ELAPSED_TIME/60))m$(($ELAPSED_TIME%60))s"
 
 if [ "$KEEP" == "0" ]; then
-	rm -rf "${VEP_OUTPUT_DIR}"
+  rm -rf "${ANNOTATE_OUTPUT_DIR}"
 fi
 
-cp "${GATK_OUTPUT}" "${OUTPUT}"
+cp "${FILTER_OUTPUT}" "${OUTPUT}"
```

Comment:
how about: moving and creating a symbolic link in the intermediate output folder? scales better for large outputs. same for report.